homeassistant/components/elevenlabs/__init__.py

```diff
@@ -4,8 +4,7 @@
 
 from dataclasses import dataclass
 
-from elevenlabs import Model
-from elevenlabs.client import AsyncElevenLabs
+from elevenlabs import AsyncElevenLabs, Model
```

Comment:
The `AsyncElevenLabs` client can now be imported from `elevenlabs.__init__` as well.